prometheus/utils.go

```diff
@@ -35,7 +35,7 @@ func connectRaw(_ context.Context, cm *connection.Manager, c *plugin.Connection)
 	// Error if the minimum config is not set
 	if address == "" {
 		// Panic since we cannot create a valid empty API to return
-		panic("address must be configured")
+		panic("Prometheus server address must be configured!")
```

Comment:
```suggestion
		panic("address must be configured")
```

I think it's good to keep the arg name in the message as is since the plugin name is included in the Steampipe error message, e.g., `Warning: failed to start plugin 'prometheus': address must be configured`